lib/src/meta.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Linter metadata annotations.
```

Comment:
nit: You need a directive in order to differentiate between library docs and docs on a member.

Reply:
Yes.   I suppose this is dangling.  (Call for another warning perhaps?  🤔)

I thought I recalled library directives falling out of favor...  Besides cooking up a library name, is there another preferred convention?

Reply:
> I thought I recalled library directives falling out of favor ...

They are. Any directive will do, but in a library like this one that doesn't need any directives I don't know what else to do. It seems silly to add a `import 'dart:core';` to avoid adding a library directive.